dependencies.yaml

```diff
@@ -106,6 +106,7 @@ dependencies:
         packages:
           - c-compiler
           - cxx-compiler
+          - libcurl>=7.87.0  # Need <https://curl.se/libcurl/c/CURLOPT_WRITEFUNCTION.html>
```

Comment:
That manpage says WRITEFUNCTION was added in curl 7.1 ?

Reply:
need `CURL_WRITEFUNC_ERROR`, add: [ecd3311](https://github.com/rapidsai/kvikio/pull/464/commits/ecd3311ad58da6f18def68b10db5a3d1ec8e150c)